lib/components/atoms/Video/Video.js

```diff
@@ -0,0 +1,63 @@
+// @flow
+/**
+ *
+ * Video
+ *
+ */
+import React from 'react';
+import styled from 'styled-components';
+
+import styles from './Video.style';
+import Track from '../Track';
+import type { Props } from './types';
+
+const Video = ({
+  width,
+  height,
+  autoPlay,
+  controls,
+  loop,
+  muted,
+  poster,
+  src,
+  children,
+  className,
+  controlsList,
+  defaultCaption,
+}: Props): Node => {
+  return (
+    <video
+      width={width}
+      height={height}
+      controls={controls}
+      autoPlay={autoPlay}
+      loop={loop}
+      muted={muted}
+      poster={poster}
+      className={className}
+      controlsList={controlsList}
+    >
+      <source src={src} type={`video/${src.substring(src.lastIndexOf('.') + 1)}`} />
+      {children.length > 0 ? children : <Track {...defaultCaption} />}
+    </video>
+  );
+};
+
+Video.defaultProps = {
+  width: '',
```

Comment:
Use a default value, most likely all your videos be 100% to their container. May be its better to preserve a aspect ratio through CSS.

Reply:
as we are not passing any value for width So, width will be automatically expandable with their ancestor So no need to write any css for this particular container. 
If there will any need to add the css then we have class prop and then we can decide the values there.